observe_configure_script.sh

```diff
@@ -883,6 +883,8 @@ EOF
     #####################################
     rhel|centos)
       log "RHEL OS"
+
+      releasever=$(grep VERSION_ID /etc/os-release | cut -d '=' -f2 | tr -d '"')
```

Comment:
This is not needed. `releasever` is a yum variable that [is already available from the OS](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/deployment_guide/sec-using_yum_variables). 
When I ran into the issue, I found that we were [not escaping this variable in the shell script](https://github.com/observeinc/linux-host-configuration-scripts/blob/b87212f76b73a64a579181df3ea3d9718a138403/observe_configure_script.sh#L940), so an empty value was written to `/etc/yum.repos.d/`. We should not substitute this variable ourselves at any point.
Changing `baseurl = https://packages.fluentbit.io/centos/$releasever/` to `baseurl = https://packages.fluentbit.io/centos/\$releasever/` should do, and then `yum install` can resolve its internal `$releasevar` variable.